src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -249,12 +249,13 @@ public  boolean supports(Class<? extends ICapability> capability) {
 	
 	@SuppressWarnings("unchecked")
 	@Override
-	public <T extends ICapability> void accept(CapabilityVisitor<T> visitor){
+	public <T extends ICapability, R> R accept(CapabilityVisitor<T, R> visitor, R unsupportedCapabililityValue){
```

Comment:
just so that I understand the API: what is the aim of unsupportedCapabilityValue?